resources/js/svir_viewer_d3_tree.js

```diff
@@ -32,6 +33,8 @@
             position: {my: "left top", at: "left top", of: "#projectDefDialog"},
             autoOpen: false,
             modal: true,
+            dialogClass: "no-close",
+            closeOnEscape: false,
```

Comment:
The idea was to bind escape to a "cancel" function, more complex than just closing the dialog. I have to check, though, if we actually did it.